app/models.py

```diff
@@ -37,10 +50,11 @@ def print_board(self):
         print(full_board)
 
     def is_valid_move(self, move: PieceMove) -> bool:
-        pass
+        return 1
```

Comment:
I think what you meant here is:
```suggestion
        return bool(1)
```

It's true that:
```python
bool(0) # false
bool(1) # true
```

but let's just return the actual value :)
```suggestion
        return True
```

Reply:
yes sure